core/src/avm2/globals/flash/display/displayobject.rs

```diff
@@ -915,6 +916,50 @@ fn global_to_local<'gc>(
     Ok(Value::Undefined)
 }
 
+fn get_bounds<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    if let Some(dobj) = this.and_then(|this| this.as_display_object()) {
+        // TODO: add typing `(target: DisplayObject)` for proper type errors
+        if let Some(target) = args.get(0).cloned().and_then(|value| match value {
+            Value::Undefined | Value::Null => Some(dobj),
+            _ => value.as_object().and_then(|o| o.as_display_object()),
+        }) {
+            let bounds = dobj.bounds();
+            let out_bounds = if DisplayObject::ptr_eq(dobj, target) {
```

Comment:
Upon reading that comment
```
            // Transform AABB to target space.
            // Calculate the matrix to transform into the target coordinate space, and transform the above AABB.
            // Note that this doesn't produce as tight of an AABB as if we had used `bounds_with_transform` with
            // the final matrix, but this matches Flash's behavior.
```
There's a possibility AVM2 `get_bounds` does it the more precise way, would need to check. For now, do you think this can be merged as-is? Will refactor next week either way.

Reply:
Yeah, sounds good.